examples/kubernetes/glusterd-ds.yaml

```diff
@@ -18,69 +18,69 @@ spec:
     spec:
       hostNetwork: true
       containers:
-      - image: docker.io/gluster/glusterd2-nightly
-        imagePullPolicy: IfNotPresent
-        name: glusterfs
-        resources:
-          requests:
-            memory: 100Mi
-            cpu: 100m
-        volumeMounts:
+        - image: docker.io/gluster/glusterd2-nightly
```

Comment:
Github did a terrible job w/ the diff of this file... It's just an indentation change (of quite a few lines):
```
foo
- a list
```
to
```
foo
  - a list
```